code/modules/mob/animations.dm

```diff
@@ -191,15 +191,17 @@ note dizziness decrements automatically in the mob's Life() proc.
 	animate(src, pixel_x = pixel_x + pixel_x_diff, pixel_y = pixel_y + pixel_y_diff, transform = rotated_transform, time = 2, easing = BACK_EASING | EASE_IN)
 	animate(pixel_x = default_pixel_x, pixel_y = default_pixel_y, transform = initial_transform, time = 2, easing = SINE_EASING)
 
-/mob/do_attack_animation(atom/A, var/atom/attack_item)
+/mob/do_attack_animation(atom/A, var/atom/attack_item, var/image/attack_image)
```

Comment:
Add a comment documenting that you should use either attack_item or attack_image